Fixed traceback when connecting to discovered printer's server.

	* cupshelpers.py (Printer.update): Work around pycups bug with
	printer-uri-supported.

Index: system-config-printer-1.0.0/cupshelpers.py
===================================================================
--- system-config-printer-1.0.0.orig/cupshelpers.py	2008-06-22 18:17:41.446820476 +0200
+++ system-config-printer-1.0.0/cupshelpers.py	2008-06-22 18:17:45.478835299 +0200
@@ -68,6 +68,8 @@ class Printer:
         self.make_and_model = kw.get('printer-make-and-model', "")
         self.type = kw.get('printer-type', 0)
         self.uri_supported = kw.get('printer-uri-supported', "")
+        if type (self.uri_supported) == list:
+            self.uri_supported = self.uri_supported[0]
         self._expand_flags()
         if self.is_shared is None:
             self.is_shared = not self.not_shared
